digiwf-integrations/digiwf-address-integration/digiwf-address-integration-service/src/test/resources/application.yml

```
@@ -12,9 +13,7 @@ spring:
                 integrationTestConsumer-in-0:
                     destination: "dwf-connector-e2e-test"
                     group: dwf-address-service-e2e
-            function:
-                routing:
-                    enabled: true
+
```

```suggestion
            function:
                routing:
                    enabled: true
```

Das function routing muss hier bleiben. Zumindest stehts so in der Doku (https://github.com/spring-cloud/spring-cloud-stream/blob/main/docs/modules/ROOT/pages/spring-cloud-stream/event-routing.adoc#routing-to-consumer)